src/api/test/unit/package_remove_test.rb

```diff
@@ -0,0 +1,119 @@
+# Testing the things that should and shouldn't happen when you remove a Package
+require File.expand_path(File.dirname(__FILE__) + "/..") + "/test_helper"
+
+class PackageRemoveTest < ActiveSupport::TestCase
+
+  fixtures :all
+
+  def setup
```

Comment:
I know what the variable does :smiley_cat:  What I don't understand is that if you call `start_test_backend` in `setup` it is not set... :-)